### PR TITLE
new trial to fix issue 2476 without changing existing option behavior

### PR DESCRIPTION
**Pull request by roccojonack**
I added a new flag and a new test perl script. I think this avoids any change in the existing use model and only adds a new switch which provides what I (and some people I know) would expect.

### CODE REVIEW

**Comment by roccojonack** [comment]
I added a new flag and a new test perl script. I think this avoids any change in the existing use model and only adds a new switch which provides what I (and some people I know) would expect.

Added also name in contributor list.

**Comment by roccojonack** [comment]
I have no preference there. If you prefer that behavior, I will follow your lead...

**Comment by wsnyder** [comment]
Yes, would prefer the 512 as a number, thanks

File: bin/verilator [hunk 1]
```diff
@@ -363,6 +363,7 @@ arguments.
     --pins-bv <bits>            Specify types for top level ports
     --pins-sc-uint              Specify types for top level ports
     --pins-sc-biguint           Specify types for top level ports
+    --pins-sc-biguint-nolimit   Specify types for top level ports
```
Comment (wsnyder) [inline bin/verilator:366]: I think this needs to be updated, you used --pins-limit, but I think what you had here makes more logical sense without the "no" that is "--pins-sc-buguint-limit", so please update the flag to that, then I think it's good to merge.  Thanks.

**Comment by wsnyder** [comment]
@roccojonack perhaps you would like to make the review change so we can get this finished off and merged in?

**Comment by wsnyder** [comment]
Also resolve conflicts as got stale, please.

**Comment by wsnyder** [comment]
Looks like this has gone stale, if you'd like to rebase and get it in please feel free to reopen.

**Pull request closed without merging**